src/py/flwr/server/driver/grpc_driver.py

```diff
@@ -48,11 +48,18 @@
 
 from .driver import Driver
 
-ERROR_MESSAGE_DRIVER_NOT_CONNECTED = """
-[flwr-serverapp] Error: Not connected.
+ERROR_MESSAGE_PUSH_MESSAGES_RESOURCE_EXHAUSTED = """
+[Driver.push_messages] gRPC error occurred:
 
-Call `connect()` on the `GrpcDriverStub` instance before calling any of the other
-`GrpcDriverStub` methods.
+The 2GB gRPC limit has been reached. Consider reducing the number of messages pushed 
+in the batch or sample a smaller number of nodes before pushing the messages.
+"""
+
+ERROR_MESSAGE_PULL_MESSAGES_RESOURCE_EXHAUSTED = """
+[Driver.pull_messages] gRPC error occurred:
+
+The 2GB gRPC limit has been reached. Consider reducing the number of messages pulled 
+in the batch or sample a smaller number of nodes before pulling the messages.
```

Comment:
We won't sample nodes before pulling messages.

```suggestion
The 2GB gRPC limit has been reached. Consider reducing the number of messages pulled 
in the batch before pulling the messages.
```